InstagramKitDemo/InstagramKitDemo/Classes/View/IKMediaViewController.m

```diff
@@ -37,7 +37,7 @@ - (void)viewDidLoad
     [super viewDidLoad];
 	// Do any additional setup after loading the view.
     self.title = [NSString stringWithFormat:@"@%@",self.media.user.username];
-    [self testLoadCounts];
+    [self testLoadUserDetails];
```

Comment:
Note that I have included the fix proposed by #96 that resolved issue #94  that prevented the demo project of building.